### ISSUE TITLE
When loading a project on weak VM IDE load is getting stuck.

### ISSUE THREAD

**Issue by sylwiabr** [enso-org/enso#7288]
### What type of issue is this?

Permanent – Occurring repeatably

### Is this issue blocking you from using Enso?

- [ ] Yes, I can't use Enso because of this issue.

### Is this a regression?

- [ ] Yes, previous version of Enso did not have this issue.

### What issue are you facing?

On a VM (4 core AMD EPYC VM with 16GB RAM) when running locally installed Enso with a local project the IDE load pause. Bringing dev tools unblocks it.

<img width="524" alt="image" src="https://github.com/enso-org/enso/assets/12892578/5d208caa-35fb-4ae6-9d03-9916fbb176dd">

### Expected behaviour

IDE should load without problems

### Enso Version

Nightly 11.07

### Browser or standalone distribution

Standalone distribution (local project)

### Browser Version or standalone distribution

standalone

### Operating System

MacOS

**Comment by JaroslavTulach** [comment]
The following message 

> Bringing dev tools unblocks it.

seems to suggest this is IDE related issue. But I am not 100% sure. Anyway adding the `-gui` label. Sylwia, if you suspect the problem is on engine/ls side, then we need deeper information:
- #6775

**Comment by kazcw** [comment]
To freeze during before main entry points, it has to be a GUI problem, probably at the JS/electron level. Electron does not run the IDE when the window is not considered in the foreground, and that's probably the only way before main entry points could take 70s.

**Comment by kazcw** [comment]
@sylwiabr, just checking about "Permanent – Occurring repeatably"--I'm surprised about that:
- Does it happen every time in this VM?
- Is it always around 70s, or is there a lot of variation?

**Comment by sylwiabr** [comment]
The 70 s is actually how long we waited with opening the console - this is this exact time. And yes, this is happening every time in this VM AFAIR (@jdunkerley please confirm)

**Comment by farmaazon** [comment]
Closing, as new GUI2 does not have "before main entry points". If any problems will be spotted on weak machines, let's open a new issue.
